src/components/solverComponents/SAT_Components/prism-SAT.js

```diff
@@ -0,0 +1,13 @@
+export const SAT_language = {
+    'SAT-operator': {
+        pattern: /\b(?:and|or|AND|OR)\b/,
```

Comment:
can we allow operator signs like `&`, `|`, `+` and `*`?

Reply:
added &, ! and | ,  what are + and * supposed to be?

Reply:
added both